src/FEValues/GeometryValues.jl

```diff
@@ -0,0 +1,135 @@
+struct MappingValues{JT, HT<:Union{Nothing,AbstractTensor{3}}}
+    J::JT # dx/dξ # Jacobian
+    H::HT # dJ/dξ # Hessian
+end
+@inline getjacobian(mv::MappingValues) = mv.J 
```

Comment:
An accessor would not be possible since `MappingValues` isn't stored. 
But it would be possible to have `calculate_jacobian(cv, x)`, but this would have to perform the calculation. Alternatively `calculate_mapping(cv, x)` would also be possible to return an instance of `MappingValues`.

Reply:
Perhaps it should be stored? The expensive part is computing it right?

Reply:
In most cases the function mapping is equally or more expensive. 
And we cannot reuse these values internally

Reply:
Since I cannot find our discussion I will put it here. We might want to provide a function to compute J inverse. For standard elements it is actually as easy as computing `inv(J)`, but for embedded elements we need `pinv(J)` I think.

With this it would be nice to propagate the mapping information into https://github.com/Ferrite-FEM/Ferrite.jl/blob/e9ae940a3b6965d78bfd1ae03827e7584bfddac4/src/PointEval/PointEvalHandler.jl#L118-L141 as a better abstraction layer (and to allow embedded elements to be found :)).

Reply:
There is `calculate_Jinv` for that purpose, but regarding making this search work for embedded elements: If ok, I would like to leave that out of this PR as it already contain too many changes in one go IMO.